modape/scripts/modis_download.py

```diff
@@ -72,6 +72,7 @@ def cli(products: List[str],
         multithread (bool): Use multiple threads for downloading.
         nthreads (int): Number of threads for multithread.
         collection (str): MODIS collection version.
+        :rtype: list
```

Comment:
Sorry, one more thing I've realized:

For the docs, the return should be following the google schema:
```python

Returns:
        List of results returned by CMR API for query


```

Reply:
Do I need to copy-paste that in my code editor? I am looking for an "Accept" button ;-)

Reply:
Lol, yeah sorry. Copy and paste. But I give you the creative liberty to re-phrase. 

I would need to clone the branch and then add and push. I figured if you add it it's easier

Reply:
Well, I copy/pasted from PR thread into modis_download.py

Reply:
> Lol, yeah sorry. Copy and paste. But I give you the creative liberty to re-phrase.
> 
> I would need to clone the branch and then add and push. I figured if you add it it's easier

Sure. I just did not expect it was going to be an archaic workflow as copy/pasting

Reply:
Sorry, I don't want to nitpick, but I didn't come up with the standards - and the automatic docbuilder might be failing.

Can you add a linebreak between the args and `Returns`?

Here's for copy-paste: :wink:  

```python
Args:
        products (List[str]): List of MODIS product codes to download.
        begin_date (datetime.date): Start date for query / download.
        end_date (datetime.date): End date for query / download.
        targetdir (pathlib.Path): Target directory.
        roi (str): Region of interest (point or bbox as csv string).
        tile_filter (str): MODIS tile filter (as csv string of tile IDs).
        username (str): Earthdata username.
        password (str): Earthdata password.
        strict_dates (bool): Strict date handling.
        download (bool): Download data.
        overwrite (bool): Replace existing.
        multithread (bool): Use multiple threads for downloading.
        nthreads (int): Number of threads for multithread.
        collection (str): MODIS collection version.
    
Returns:
        List of results returned by CMR API for query
```